cheroot/server.py

```diff
@@ -87,7 +87,7 @@
 from . import errors, __version__
 from ._compat import bton, ntou
 from .workers import threadpool
-from .makefile import MakeFile
+from .makefile import MakeFile, StreamWriter
```

Comment:
*(It annoys me that I've failed to remove `MakeFile` at all, but that's all I can do for now)*